@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -934,6 +946,12 @@ export interface DashedLineTechniqueParams extends BaseTechniqueParams, Polygona
      * Clip the line outside the tile if `true`.
      */
     clipping?: boolean;
+
```

Comment:
Out of scope for your topic, but I think `DashedLineTechniqueParams` should extend `SolidLineTechniqueParams`

Reply:
It makes sense to me. Now there are other pool requests that touch this code.
https://github.com/heremaps/harp.gl/pull/754
It is possible to do this upon completion of this task.
Please specify what to do next.